.travis.yml

```diff
@@ -1,7 +1,8 @@
 sudo: false
 language: ruby
 before_install:
-  - gem update bundler
+  -  gem uninstall bundler
+  -  gem install bundler -v '1.17.3'
```

Comment:
The changelog is missing to describe your intension ;)

Reply:
Fixed